src/profile/components/forms/Certificates.jsx

```diff
@@ -50,24 +49,25 @@ class Certificates extends React.Component {
   }
 
   renderCertificate({
-    certificateType, courseDisplayName, courseOrganization, modifiedDate, downloadUrl,
+    certificateType, courseDisplayName, courseOrganization, modifiedDate, downloadUrl, courseId,
   }) {
     const { intl } = this.props;
-    const certificateIllustration = ((type) => {
-      switch (type) {
-        case 'Professional Certificate':
+    const certificateIllustration = (() => {
+      switch (certificateType) {
+        case 'professional':
+        case 'no-id-professional':
           return professionalCertificateSVG;
-        case 'MicroMasters Certificate':
-          return microMastersSVG;
```

Comment:
No more MM SVG?

Reply:
We don't currently display any program certificates. At the moment this case would never be used.